cli/src/pcluster/validators/ec2_validators.py

```diff
@@ -150,6 +150,7 @@ def _validate(self, instance_type: str, image: str):
 
         if (
             image_info
+            and image_info.description
```

Comment:
is it possible to add an unit test to cover the issue?

Reply:
Yes, Done